gabbi/runner.py

```diff
@@ -10,7 +10,7 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-"""Implementation of a command-line runner of single gabbi files."""
+"""Implementation of a command-line runner of single Gabbi files."""
```

Comment:
You appear to accidentally capitalised gabbi where it wasn't before. I assume this was a mistake of some kind?